### DISCUSSION TITLE
FD and DVARs confound files

### DISCUSSION THREAD

**Discussion by dsodums** [McIntosh-Lab/tvb-ukbb#178] [Help]
Dear experts,

We are looking into using the FD and DVARs .txt files obtained for each subject in the Rockland dataset, to apply some post processing motion correction. I see in addition to the DVARS_rfMRI_0.ica.txt and FD_rfMRI_0.ica.txt files, there are the following files as well:

DVARS_confound_mat_rfMRI_0.ica.txt
FD_confound_mat_rfMRI_0.ica.txt

Could you please provide a brief description of what these confound files are?

Thanks,
Devin

**Comment by justinliuyuwang** [comment]
Hi Devin,

These command used to generate confound files is described here: https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLMotionOutliers
There should be a brief description of the confound files there.

Hope that helps!
Justin
